src/pages/personnel/duck/Sagas.js

```diff
@@ -0,0 +1,52 @@
+import {
+    call,
+    put,
+    takeLatest,
+} from "redux-saga/effects";
+import axios from "axios";
+
+import errorSelectors from "components/error/duck/Selectors";
+import errorTypes from "components/error/duck/Types";
+import types from "./Types";
+
+function* load() {
+    try {
+        const { data } = yield call( axios.get, "/personnel/" );
+        yield put( { type: types.LOAD_SUCCESS, payload: data } );
+    } catch ( error ) {
+        yield put( {
+            type: errorTypes.APP_ERROR,
+            payload: errorSelectors.errorPayload( error ),
+        } );
+    }
+}
+
+function* create( { payload } ) {
+    try {
+        const { data } = yield call( axios.post, "/personnel/", payload );
+        yield put( { type: types.CREATE_SUCCESS, payload: data } );
+    } catch ( error ) {
+        yield put( {
+            type: errorTypes.APP_ERROR,
```

Comment:
Use the error selector here (to display validation errors)

Reply:
Bij create of?? Want die erboven gebruikt em gwn. Zit op mobiel dus zie niet alles